lib/membrane/srtcp/decryptor.ex

```diff
@@ -0,0 +1,53 @@
+defmodule Membrane.SRTP.Decryptor do
+  @moduledoc """
+  Converts SRTP packets to plain RTP.
+
+  Requires adding [srtp](https://github.com/membraneframework/elixir_libsrtp) dependency to work.
+  """
+  use Membrane.Filter
+
+  alias Membrane.Buffer
+
+  def_input_pad :input, caps: :any, demand_unit: :buffers
+  def_output_pad :output, caps: :any
+
+  def_options policies: [
+                spec: [LibSRTP.Policy.t()],
+                description: """
+                List of SRTP policies to use for decrypting packets.
+                See `t:LibSRTP.Policy.t/0` for details.
+                """
+              ]
+
+  @impl true
+  def handle_init(options) do
+    {:ok, Map.from_struct(options) |> Map.merge(%{srtp: nil})}
```

Comment:
The name could be more meaningful, we often used `native`

Reply:
Since the native is hidden behind LibSRTP, I believe srtp is meaningful, while native wouldn't be